src/context_diff.rs

```diff
@@ -265,12 +265,33 @@ fn make_diff(
     results
 }
 
+fn get_modification_time(file_path: &str) -> String {
```

Comment:
This new helper function should probably go to utils.rs. The unified diff format also displays timestamps, so unified_diff.rs also needs to use it.